bin/github-backup

```diff
@@ -537,6 +537,24 @@ def _request_url_error(template, retry_timeout):
     return False
 
 
+def download_file(url, path, auth):
+    request = Request(url)
+    request.add_header('Accept', 'application/octet-stream')
```

Comment:
It's also worth noting, the request succeeds when removing this header (you just get the JSON response instead of the binary data).